kuma/attachments/signal_handlers.py

```diff
@@ -5,7 +5,7 @@
 
 
 @receiver(pre_delete, sender=AttachmentRevision,
```

Comment:
The signals need to be swapped as well, so this should be `post_delete` instead of `pre_delete`.